src/autopas/AutoPasDecl.h

```diff
@@ -215,6 +240,30 @@ class AutoPas {
   const_iterator_t getRegionIterator(std::array<double, 3> lowerCorner, std::array<double, 3> higherCorner,
                                      IteratorBehavior behavior = IteratorBehavior::ownedOrHalo) const;
 
+  /**
+   * execute code on all particles in a certain region as defined by a lambda function
+   * @tparam Lambda (Particle &p) -> void
+   * @param forEachLambda code to be executed on all particles
+   * @param lowerCorner lower corner of bounding box
+   * @param higherCorner higher corner of bounding box
+   * @param behavior @see IteratorBehavior default: @see IteratorBehavior::ownerOrHalo
+   */
+  template <typename Lambda>
+  void forEachInRegion(Lambda forEachLambda, std::array<double, 3> lowerCorner, std::array<double, 3> higherCorner,
+                       IteratorBehavior behavior = IteratorBehavior::ownedOrHalo) {
+    _logicHandler->forEachInRegion(forEachLambda, lowerCorner, higherCorner, behavior);
```

Comment:
@FG-TUM 
Is there a way to define these in `AutoPasImpl` and remove the imports of `LogicHandler` and `AutoTuner` from `AutoPasDecl` again.